src/main/java/pl/damianszczepanik/jenkins/buildhistorymanager/model/actions/DeleteLogFileAction.java

```diff
@@ -15,16 +15,30 @@
  */
 public class DeleteLogFileAction extends Action {
 
+    private static final Logger LOG = Logger.getLogger(DeleteLogFileAction.class.getName());
+    
     @DataBoundConstructor
     public DeleteLogFileAction() {
         // Jenkins stapler requires to have public constructor with @DataBoundConstructor
     }
 
     @Override
     public void perform(Run<?, ?> run) throws IOException, InterruptedException {
-        File logFile = run.getLogFile();
+        File logFile;
+        try {
+            logFile = run.getLogFile();
+        } catch (UnsupportedOperationException e) {
+            log(run.getParent().getFullDisplayName(), "accessing log as file is unsupported");
```

Comment:
First of all this message is not clear to non technical Jenkins user, even for me might be hard to understand without context.
Second I'm not sure if this should result as log or something that tell users that configuration must be updated

Reply:
What so you suggest instead?

Reply:
I don't have perfect idea so let's leave it as it is but for sure helper file must be updated about this behave - that should be clear for the job designer that this action has some limitation

Reply:
I hope the log message is more clear now